src/main/java/org/alliancegenome/curation_api/jobs/executors/BulkLoadJobExecutor.java

```diff
@@ -138,6 +140,8 @@ public void process(BulkLoadFileHistory bulkLoadFileHistory, Boolean cleanUp) th
 			expressionAtlasExecutor.execLoad(bulkLoadFileHistory);
 		} else if (bulkLoadFileHistory.getBulkLoad().getBackendBulkLoadType() == BackendBulkLoadType.BIOGRID_ORCS) {
 			biogridOrcExecutor.execLoad(bulkLoadFileHistory);
+		} else if(bulkLoadFileHistory.getBulkLoad().getBackendBulkLoadType() == BackendBulkLoadType.HTPDATASAMPLE) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.
<!-- __reviewdog__:ChAzOGI3MzQyYWNiMmNjMDBlEgpjaGVja3N0eWxl -->